third_party/grpc-rust/protoc-rust-grpc/src/lib.rs

```diff
@@ -133,9 +134,9 @@ fn remove_path_prefix<'a>(mut path: &'a str, mut prefix: &str) -> Option<&'a str
     }
 
     if path.as_bytes()[prefix.len()] == b'/' || path.as_bytes()[prefix.len()] == b'\\' {
-        Some(&path[prefix.len() + 1..])
+        return Some(&path[prefix.len() + 1..]);
```

Comment:
Is this your change or does it come from upstream?

Reply:
All changes are from upstream